code/modules/antagonists/traitor/medrp/new_objectives.dm

```diff
@@ -0,0 +1,34 @@
+
+
+/obj/effect/proc_holder/spell/self/add_objectives //If all of your objectives are done, you get two more.
+	name = "Request More Objectives"
```

Comment:
At the moment I liked how sticky spells were in staying attached to the character (aka it follows mind, not mob) but actually a way better idea is making this a button in an uplink.